### PR TITLE
Add possibility to init with custom host

### PR DESCRIPTION
**Pull request by michalpaukert**

### CODE REVIEW

**Comment by michalpaukert** [comment]
There is currently no possibility to use custom host, I want be able to set custom host.

**Comment by stefanjudis** [comment]
That looks great! :👏

@michalpaukert could we reflect these changes in the readme somewhere? :)

**Comment by michalpaukert** [comment]
@stefanjudis I made requested changes, could you please go throw and accept it?

File: README.md [hunk 1]
```diff
@@ -78,6 +78,21 @@ If you don't provide any (or fields by your strategy don't exist or don't have a
...
+## Initialization
+
+> In default configuration there is pre-configured host for you published entries and assets.
+> If you need work with preview items or custom host you have to set in you init script following.
+
+```js
+CTFLWizard.init({
+  spaceId: 's25qxvg',
+  key: 'f78aw812mlswwasw', // your API key
+  host: 'preview.contentful.com' //Preview host
```
Comment (stefanjudis) [inline README.md:91]:
```suggestion
  host: 'preview.contentful.com' // preview host
```

File: README.md [hunk 2]
```diff
@@ -78,6 +78,21 @@ If you don't provide any (or fields by your strategy don't exist or don't have a
 
 > There is no "smart" guessing strategy (like trying to list all fields with string value and get one with a short enough value), since it can easily introduce incosistency, and it will become confusing.
 
+
+## Initialization
+
+> In default configuration there is pre-configured host for you published entries and assets.
+> If you need work with preview items or custom host you have to set in you init script following.
```
Comment (stefanjudis) [inline README.md:85]:
```suggestion
> If you need to work with preview items define a custom host as `init` parameter.
```

**Review by stefanjudis — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**